File: Selenium/Activities/activity11.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class activity11 {
    public static void main(String args[]) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/selenium/DynamicControls");
        System.out.println("Home page title: " + driver.getTitle());
        WebElement checkbox = driver.findElement(By.xpath("//input[@id='checkbox']"));
        checkbox.click();
        System.out.println("Checkbox is selected: " + checkbox.isSelected());
        // Click the checkbox to deselect it
        checkbox.click();
        System.out.println("Checkbox is selected: " + checkbox.isSelected());
        driver.close();
    }
}

File: Selenium/Activities/activity11.py
from selenium.webdriver.common.by import By
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
driver = webdriver.Firefox(service=FirefoxService(GeckoDriverManager().install()))
driver.get("https://training-support.net/selenium/DynamicControls")
print("Page title is: ", driver.title)
checkbox = driver.find_element(By.XPATH, "//input[@id='checkbox']")
checkbox.click();
print("Checkbox is selected: ", checkbox.is_selected())
checkbox.click()
print("Checkbox is selected: ", checkbox.is_selected())
driver.close()
